driver/src/order_driver.rs

```diff
@@ -85,8 +80,8 @@ impl<'a, D: DbInterface, C: SnappContract> OrderProcessor<'a, D, C> {
                     }
                     return Err(DriverError::new(
                         &format!("Cannot find saved bid for auction slot {:?}", slot),
-                        ErrorKind::StateError
-                    ))
+                        ErrorKind::StateError,
+                    ));
```

Comment:
How is it even possible that we did not have a semi-colon here before?

Reply:
I guess for return statements, you don't need to have a `;`